src/components/search/search.test.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import {
+  render,
+  fireEvent,
+  screen,
+  getByLabelText,
+  getAllByTestId,
+} from "@testing-library/react";
+import "@testing-library/jest-dom";
+import Search from "@/components/search/search";
+import { useRouter } from "next/router";
+
+jest.mock("next/router", () => ({
+  useRouter: jest.fn(),
+}));
+const mockRouter = {
+  push: jest.fn(),
+};
+(useRouter as jest.Mock).mockReturnValue(mockRouter);
```

Comment:
This is fine and I'm definitely NOT saying you need to change this. Just wanted to show you what Diego/Vera have done recently with the `next-router-mock` package: https://github.com/NYPL/research-catalog/blob/main/__test__/pages/search/searchResults.test.tsx